packages/hawtio/src/plugins/camel/routes-service.ts

```diff
@@ -20,6 +20,55 @@ export type CamelRoute = {
   MeanProcessingTime: number
 }
 
+export type ProcessorStats = {
+  id: string
+  index: number | string
+  state: string | null
+  sourceLineNumber: number | string
+  exchangeInflight: number | string
+  exchangesCompleted: number | string
+  redeliveries: number | string
+  externalRedeliveries: number | string
+  minProcessingTime: number | string
+  maxProcessingTime: number | string
+  deltaProcessingTime: number | string
+  meanProcessingTime: number | string
+  startTimestamp: number | string
+  resetTimestamp: number | string
+  firstExchangeCompleteTimestamp?: number | null
+  firstExchangeCompletedExchangeId?: number | null
+  lastExchangeCompleteTimestamp?: number | null
```

Comment:
it doesn't seem to make sense to have it optional and type `number | null` at the same time. can't you simply `lastExchangeCompleteTimestamp?: number`?

same for all other variables.